keyboards/handwired/co60/rev6/rules.mk

```diff
@@ -0,0 +1,60 @@
+# project specific files
+LAYOUTS += all 60_ansi 60_ansi_split_bs_rshift 60_iso 60_hhkb 60_hhkb_split_space 60_hhkb_split_625u_space
```

Comment:
Could you move this to the end? 

It's not necessary to do so, but in general, that's how most are formatted.